ojdbc-provider-jackson-oson/src/main/java/oracle/jdbc/provider/oson/OsonGenerator.java

```diff
@@ -77,7 +76,9 @@ public class OsonGenerator extends GeneratorBase {
   /**
    * Constructs an OsonGenerator with the specified features, codec, OracleJsonGenerator,
    * and output stream.
-   *
+   * <p>
+   * Note that the underlying <code>OutputStream</code> will not hold any data until the close method is called on this instance
+   * @see #close()
    * @param features The features for the generator.
```

Comment:
Shouldn't we add the note on the flush method as well. If you use the factory to create the generator you might not see then note int the constructor. It might be interesting to add the information on the factory as well.

Reply:
Where is that exactly ?